gmplot/gmplot.py

```diff
@@ -45,22 +45,22 @@ def geocode(self, location_string):
     def grid(self, slat, elat, latin, slng, elng, lngin):
         self.gridsetting = [slat, elat, latin, slng, elng, lngin]
 
-    def marker(self, lat, lng, color='#FF0000', c=None, title="no implementation"):
+    def marker(self, lat, lng, title="no implementation", color='#FF0000', c=None):
         if c:
             color = c
         color = self.color_dict.get(color, color)
         color = self.html_color_codes.get(color, color)
         self.points.append((lat, lng, color[1:], title))
 
-    def scatter(self, lats, lngs, color=None, size=None, marker=True, c=None, s=None, **kwargs):
+    def scatter(self, lats, lngs, titles, color=None, size=None, marker=True, c=None, s=None, **kwargs):
```

Comment:
Let's move `titles` to the end (right before `kwargs`) as an optional parameter that defaults to `None`, to preserve the existing parameter order.